content/en/getting_started/synthetics/api_test.md

```diff
@@ -22,60 +25,79 @@ further_reading:
 
 ### Configure the request
 
 1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetic Test**.
 2. In the top right corner, click the **New Test** button.
 3. Select **API test**.
-4. Define the configuration of your API test:
+4. Define your API request:
 
-    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, which is a test web application.
-    - Select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
-    - You can set tags such as `env:prod` and `app:shopist` on your test. Tags help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
-    - Choose from the **Managed Locations** to run your test from.
-    - Click the **Test URL** button.
+    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, a test web application.
+    - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - You can set environment tags such as `env:prod` and `app:shopist` and additional tags on your test. Tags allow you to stay organized and quickly find test you're interested in on the homepage.
+    - Click **Test URL**.
 
 {{< img src="getting_started/synthetics/api-test-config.png" alt="API test configuration"  style="width:60%;">}}
 
-#### Define your alert conditions
+#### Define assertions
+
+Datadog automatically populates basic assertions about your endpoint's response. Assertions define the alert condition and are customizable. 
 
-After clicking Test URL, basic assertions based on your endpoint's response are automatically populated. Assertions define the alert condition and can be customized. In this example, three default assertions are populated when testing the URL:
+In this example, three default assertions populate after testing the URL:
 
 {{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure"  style="width:90%;">}}
 
-These assertions define the alert condition and can be customized. To add a custom assertion, directly click on the response preview or click the New Assertion button to add an assertion manually (e.g., body contains Shop.ist)
+To add a custom assertion, you can click anywhere in the response preview or click **New Assertion**.
 
 {{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure"  style="width:90%;">}}
 
-You can set up an alerting condition in order to only be alerted if your endpoint goes down for three minutes, on two different locations:
+#### Select locations 
+
+Select some **Managed Locations** or **Private Locations** to run your test from.
```

Comment:
```suggestion
Select one or more **Managed Locations** or **Private Locations** to run your test from.
```

Reply:
This would be a good place to have a sentence about what managed and private locations are, and where to find more info.

Reply:
@kayayarai, how's this:

Your managed locations often are public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select **Private Locations** instead.

For more information about private locations, see the [Private Locations](https://docs.datadoghq.com/getting_started/synthetics/private_location) documentation.

Reply:
I like this rewrite, but can't approve the PR until it's actually in the source. One edit to the last sentence "For more information, see the [Private Locations] documentation." (it says "PL" too many times).